src/argilla/client/feedback/training/base.py

```diff
@@ -48,7 +59,7 @@ def __init__(
 
         Args:
             dataset (FeedbackDataset): the dataset to be used for training.
-            task_mapping (TrainingTaskMappingForTextClassification): the training data to be used for training.
+            task (TrainingTask): the training data to be used for training.
```

Comment:
I feel we could just remove the type from the docstrings i.e. `task (TrainingTask):` as it's already defined in the `__init__` so it feels a little bit redundant IMO, thoughts? @tomaarsen @gabrielmbmb

Reply:
Also we're not really consistent with docstrings here and there so we should probably spend some time defining our own guidelines for docstrings 👍🏻

Reply:
I'm okay with it here. `TrainingTask` is kind of wrong anyways, as for some reason `TrainingTask` isn't a superclass of `TrainingTaskFor...`.